### PR TITLE
Added Amazon Web Service (AWS) Arabic Playlist

### PR DESCRIPTION
**Pull request by Flicker-eth**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Flicker-eth** [comment]
Greeting Sir,
Update the Link

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Comment by AhmedElTabarani** [comment]
> Thanks, and welcome to free-programming-books!

The name of the channel and the playlist still not correct here

**Comment by eshellman** [comment]
@AhmedElTabarani could you submit a PR?

**Comment by AhmedElTabarani** [comment]
done

‫في الخميس، 6 أبريل 2023 في 11:10 م تمت كتابة ما يلي بواسطة ‪Eric Hellman‬‏
***@***.***‬‏>:‬

File: courses/free-courses-ar.md
Comment (Unouwant2) [inline courses/free-courses-ar.md:1]: ``

**Pull request merged**